test/compilable/extra-files/header1i.di

```diff
@@ -186,110 +186,44 @@ class xFoo2 : iFoo, iFoo2
 }
 class Foo3
 {
-	this(int a, ...)
-	{
-	}
-	this(int* a)
-	{
-	}
+	this(int a, ...);
+	this(int* a);
 }
 alias myint = int;
 static notquit = 1;
 class Test
 {
-	void a()
-	{
-	}
-	void b()
-	{
-	}
-	void c()
-	{
-	}
-	void d()
-	{
-	}
-	void e()
-	{
-	}
-	void f()
-	{
-	}
-	void g()
-	{
-	}
-	void h()
-	{
-	}
-	void i()
-	{
-	}
-	void j()
-	{
-	}
-	void k()
-	{
-	}
-	void l()
-	{
-	}
-	void m()
-	{
-	}
-	void n()
-	{
-	}
-	void o()
-	{
-	}
-	void p()
-	{
-	}
-	void q()
-	{
-	}
-	void r()
-	{
-	}
-	void s()
-	{
-	}
-	void t()
-	{
-	}
-	void u()
-	{
-	}
-	void v()
-	{
-	}
-	void w()
-	{
-	}
-	void x()
-	{
-	}
-	void y()
-	{
-	}
-	void z()
-	{
-	}
-	void aa()
-	{
-	}
-	void bb()
-	{
-	}
-	void cc()
-	{
-	}
-	void dd()
-	{
-	}
-	void ee()
-	{
-	}
+	void a();
```

Comment:
I was expecting that in a header file you only need the function declaration. Maybe I am missing something...